INSTALL.md

```diff
@@ -26,6 +27,12 @@ For convenience, we also keep a record in the form of release tags of the
  * pre-releases are versions that get deployed to testnet
  * releases are versions that made it all the way in prod
 
+## Zero-install browser-based cloud IDE with Gitpod
```

Comment:
Agreed, `CONTRIBUTING.md` sounds like a better place, since it is precisely for learning/experimenting (and possibly deciding thereby and learning how to contribute, or maybe build upon it) that I was imagining people using this.